exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -122,29 +137,48 @@ impl CommandExtension for Finalize {
     }
 
     fn execute(&self, mut context: Context) -> Result<Context, failure::Error> {
-        let module_name = context.arg(EJB_MODULE_NAME)?;
+        let user_parameters = context
+            .arg_multiple(EJB_USER_PARAMETERS)
+            .unwrap_or_default();
+        let log_config_path = context.arg(EJB_LOG_CONFIG_PATH).unwrap_or_default();
         let port = context.arg(EJB_PORT)?;
 
-        let service_config = ServiceConfig { module_name, port };
-
-        let jvm_config: JvmConfig = context
-            .get(keys::SERVICES_SECRET_CONFIGS)
-            .expect("Can't get services secret configs")
-            .get(EJB_JVM_CONFIG_NAME)
-            .expect("Can't get JVM config")
-            .clone()
+        let mut node_config: NodeConfig = context.get(keys::NODE_CONFIG)?;
+        let mut ejb_config: Config = node_config
+            .services_configs
+            .get(EJB_CONFIG_NAME)
+            .cloned()
+            .unwrap()
             .try_into()?;
+        ejb_config.private_config.user_parameters = user_parameters;
+        ejb_config.private_config.log_config_path = log_config_path;
+        ejb_config.private_config.port = port;
 
-        let config = Config {
-            jvm_config,
-            service_config,
-        };
-
-        let mut node_config: NodeConfig = context.get(keys::NODE_CONFIG)?;
         node_config
             .services_configs
-            .insert(EJB_CONFIG_NAME.to_owned(), Value::try_from(config)?);
+            .insert(EJB_CONFIG_NAME.to_owned(), Value::try_from(ejb_config)?);
         context.set(keys::NODE_CONFIG, node_config);
         Ok(context)
     }
 }
+
+fn get_system_classpath() -> String {
+    let mut jars = Vec::new();
+    let jars_directory = {
+        let mut current_directory =
+            env::current_dir().expect("Could not get current working directory");
+        current_directory.push("lib/java");
+        current_directory
+    };
+    for entry in fs::read_dir(jars_directory).unwrap() {
+        let file = entry.unwrap();
+        if file.file_type().unwrap().is_file() {
+            jars.push(file.path());
+        } else {
+            continue;
+        }
+    }
+
+    let jars: Vec<&str> = jars.iter().map(|p| p.to_str().unwrap()).collect();
+    join_paths(&jars)
```

Comment:
Isn't it possible to use `join_paths` without an intermediate Vec? Like,

```rust
join_paths(jars.iter())
```

Reply:
Due to `join_paths` signature, not.

Reply:
Oh, I thought that was `std::env::join_paths`.

Reply:
:thinking: Why we're not using `env::join_paths`?

Reply:
vOv

`utils::join_paths` seems to have been added in #331, but I believe that's exactly what `std::env::join_paths` does.

Reply:
Got it, `utils::join_paths` is used for different purpose and can't be replaced with `std::env::join_paths`